pdd/model.py

```diff
@@ -0,0 +1,45 @@
+import numpy as np
+import torch
+import torch.nn as nn
+from torchvision.models import mobilenet_v2
+
+
+class PDDModel(nn.Module):
```

Comment:
Leave comment that model based on MobileNetV2 backbone or rename class to `PDDMobileNetV2`

Reply:
Comment should be in triple quotation marks